README.md

```diff
@@ -1,85 +1,81 @@
-DataStax Java Driver for Apache Cassandra
-=========================================
-
-.. image:: https://travis-ci.org/datastax/java-driver.svg?branch=2.0
-   :target: https://travis-ci.org/datastax/java-driver
-   :alt: Travis Build
-
+# Datastax Java Driver for Apache Cassandra
+
+*If you're reading this on GitHub, please note that this is the readme
```

Comment:
If you go to the project on GitHub, you'll still see the readme of the version currently in progress. That's what this comment is for.